lib/console.js

```
@@ -24,8 +24,14 @@ function Console(stdout, stderr) {
   Object.defineProperty(this, '_stdout', prop);
   prop.value = stderr;
   Object.defineProperty(this, '_stderr', prop);
+  prop.value = ignoreErrors === undefined ? true : !!ignoreErrors;
```

Oh, you mean to suppress the errors by default. Sorry, I missed that.